exist-core/src/main/java/org/exist/storage/NativeBroker.java

```diff
@@ -153,6 +158,34 @@
 
     private final static DigestType BINARY_RESOURCE_DIGEST_TYPE = DigestType.BLAKE_256;
 
+    protected boolean caseSensitive = true;
```

Comment:
please could you some java doc here about the semantics ?

Reply:
It seems this is not actually needed in `NativeBroker` at all, it is instead related to `NativeValueIndex` and so I have now removed it from `NativeBroker`.